client/src/Components/Layout/SideBar/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Menu, Typography, Button } from 'antd';
+import {
+  AppstoreAddOutlined,
+  UnorderedListOutlined,
+  NotificationOutlined,
+} from '@ant-design/icons';
+
+import './style.css';
+
+const { Title } = Typography;
+const { Item } = Menu;
+
+const SideBar = ({ onClick, onLogout }) => {
+  const history = useHistory();
+
+  const handleItemClick = (to) => {
+    history.push(to);
+  };
+
+  const handleLogout = () => {
+    // handle logout logic
+    onLogout();
```

Comment:
this will not work, you are trying to execute the parameter without checking its type, if you clicked on the logout without passing `onLogout` as a function will throw an Error and break down the app, and **the propTypes, not enough for validation**, I suggest you to check the type of ```logout``` before execution
```suggestion
    if( typeof onLogout === "function") {
    onLogout();
    }else {
    console.log("logout clicked !!!"
    }
```